src/pages/LandingPage/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
```

Comment:
Why would we need pages when we have containers? Having containers, are you going to add components separately? Want to know the way we are going to implement it

Reply:
Components contain just ui.
Containers contain useDispatch/useSelector calls and use Components to visualize state/actions.
Pages are navigator screens.

In most cases Page will contain just one Container, but it's good to have a "reserve" layer